src/components/whiteframe/whiteframe.html

```diff
@@ -0,0 +1 @@
+<ng-content></ng-content>
```

Comment:
I'm not sure about using a component for the `whiteframe`. Wrapping the content in a component template makes not really a sense. Thoughts?